src/NuGet.Clients/NuGet.VisualStudio.Client/.vsixinclude

```diff
@@ -34,5 +34,7 @@ NuGet.Versioning.dll
 
 # 3rd party dlls to keep
 Lucene.Net.dll
+Markdig.Signed.dll 
+Markdig.Wpf.dll
```

Comment:
I saw in another comment that other components are already using these assemblies, but there's some third party library due diligence that's required when shipping a new assembly.

Firstly, we need to modify the third party notice file in this repo with the copyright of both packages.

Secondly, check that Visual Studio's third party notices contains entries for both assemblies as well. Even if other VS components have been using the assemblies for a while, just check to make sure they did the right thing.